subiquity/models/network.py

```diff
@@ -23,7 +23,7 @@
 class NetworkModel(NetworkModel):
```

Comment:
Wait, that works?  That looks super strange.
I know this isn't part of the PR but what's your thoughts on a minor rename?

Reply:
Yeah it works fine (welcome to Python!). It might be a bit strange but can we leave it for now? :)